composer.json

```diff
@@ -43,6 +43,7 @@
     "ext-zip": "*",
     "ext-zlib": "*",
     "ext-curl": "*",
+    "cbschuld/browser.php": "^1.9.6",
```

Comment:
Browser is needed and used in https://github.com/pimcore/admin-ui-classic-bundle/blob/328928e0bd66ea549c2fd962036bd2c2a605c49f/src/Controller/Admin/LoginController.php#L411

We temporary re-added it here because when i merged this https://github.com/pimcore/pimcore/pull/14876, this would be missing from the core composer and wouldn't be installed. 
For temporary, i mean that it was only then deferred to be removed from core composer again, once the admin bundle extraction would be done